packages/react-components/react-teaching-popover/src/components/TeachingPopoverBody/useTeachingPopoverBodyStyles.styles.ts

```diff
@@ -11,28 +11,28 @@ export const useMediaStyles = makeStyles({
   base: {
     ...shorthands.gridArea('media'),
     ...shorthands.overflow('hidden'),
-    width: '288px',
+    width: '286px',
```

Comment:
Since this value seems to be re-used throughout, consider adding a semantic 'token' similar to the way I did for Nav here:

https://github.com/microsoft/fluentui/blob/ff79ac88545c9cbc7966f75f61a9d506f3ebb963/packages/react-components/react-nav-preview/src/components/sharedNavStyles.styles.ts#L7